src/Elastic.Apm.AspNetCore/Elastic.Apm.AspNetCore.csproj

```diff
@@ -2,16 +2,13 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <AssemblyName>Elastic.Apm.AspNetCore</AssemblyName>
-    <AssemblyVersion>1.0.0</AssemblyVersion>

```

Comment:
We should not have these changes - I think most of this is not up-to-date. I'd suggest just leave the `.csproj` files as they are - we have the current version on `master` and we'll increase those once we release.

Reply:
It's not out of date - I've kept it updated from the base branch. If you're referring to the change in use of version properties, this is correct (good info [here](https://andrewlock.net/version-vs-versionsuffix-vs-packageversion-what-do-they-all-mean/)). `AssemblyVersion` defaults to `Version` without any prerelease label. `FileVersion` defaults to `AssemblyVersion`. `InformationalVersion` defaults to `Version` with the prerelease label. The end result is the same, but this is the proper way to do it if you don't require specifically different values for the removed version properties other than the defaults (which isn't the case in this case).

However, I'll ensure the version is aligned with _master_ on the base branch in any case.